components/calendar/utils.ts

```diff
@@ -53,7 +39,7 @@ export const ALL_DAYS_OF_WEEK = [
   "Sun",
   "Mon",
   "Tues",
-  "Weds",
+  "Wed",
```

Comment:
If Tuesday is Tues and Thursday is Thurs, then I think Wednesday can stay as `Weds`. Otherwise I'd change to Tue, Wed, Thur